types/react/test/index.ts

```diff
@@ -408,10 +408,16 @@ interface ForwardingRefComponentProps {
     foo: number;
 }
 
-const ForwardingRefComponent = React.forwardRef((props: ForwardingRefComponentProps, ref: React.Ref<RefComponent>) => {
+const ForwardingRefComponent = React.forwardRef((props: ForwardingRefComponentProps, ref: React.ForwardedRef<RefComponent>) => {
```

Comment:
Was this change required (if not we should revert it to not cause confusion)? Just to make sure we're not causing unnecessary churn.

Reply:
Not strictly necessary. I removed it.